### PR TITLE
i have added all the basics codes for the app but still need to fix t…

### PR DESCRIPTION
**Pull request by A-Ghoujal**
…he js file

### CODE REVIEW

**Review by dakravchenko — APPROVED** [review]

**Review by YunusEmreBEYAZ — APPROVED** [review]

File: src/data.js [hunk 1]
```diff
@@ -58,3 +58,152 @@ export const quizData = {
...
+      question: " question 10 ",
+      a: " 1 ",
+      b: " 2 ",
+      c: " 3 ",
+      d: " 4 ",
+      correct: "1",
+   },
+
+];
+
```
Comment (sima-milli) [inline src/data.js]: This file is only for data, we need to move the code below to the right file, we can do that in our meeting today

File: src/data.js [hunk 2]
```diff
@@ -58,3 +58,152 @@ export const quizData = {
...
+  let answer
+  answerEls.forEach(answerEl => {
+          if(answerEl.checked){
+              answer = answerEl.id
+          }
+      })
+      return answer
+}
+
+submitBtn.addEventListener('click', () => {
```
Comment (sima-milli) [inline src/data.js]: This part can be move to the results page and views

File: src/data.js [hunk 3]
```diff
@@ -58,3 +58,152 @@ export const quizData = {
     // Add more questions here
   ],
 };
+
+
+// MY JS CODE STARTS FROM HERE ---------------------------------------------------
+
+
+
+const quizData = [
```
Comment (sima-milli) [inline src/data.js]: we need to export this variable to be able to use it in other files

File: index.html [hunk 4]
```diff
@@ -2,16 +2,43 @@
...
+        </li>
+
+        <li>
+            <input type="radio" name="answer" id="c" class="answer">
+            <label for="c" id="c_text">Answer</label>
+        </li>
 
-  <body>
-    <!-- Put your HTML here -->
-    <div id="user-interface" class="centered"></div>
```
Comment (sima-milli) [inline index.html:12]: we can keep this `div`

File: index.html [hunk 5]
```diff
@@ -2,16 +2,43 @@
...
+            <input type="radio" name="answer" id="b" class="answer">
+            <label for="b" id="b_text">Answer</label>
+        </li>
+
+        <li>
+            <input type="radio" name="answer" id="c" class="answer">
+            <label for="c" id="c_text">Answer</label>
+        </li>
 
-  <body>
```
Comment (sima-milli) [inline index.html:10]: looks like` <body>` was removed by mistake :)

**Review by sima-milli — COMMENTED** [review]
Thanks for adding the code, very good job 💪
Let's talk about the comments I added and some improvements in our meeting today
Also you need to run `npm run prettier:write` to fix the lintting issue for this PR (I can also show in the meeting)

**Review by sima-milli — APPROVED** [review]

**Pull request merged**